### PR TITLE
Network map missing ports

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/docs/issues/4107

Updated:
- Self-Managed Admin Guide > Architecture Overview > Basics
   - Added new section called **Mattermost Services Ports**

### CODE REVIEW

**Comment by cwarnermm** [comment]
@sadohert - The port details documented as part of this PR have been graciously provided by @streamer45. Can you provide a second set of eyes as a technical reviewer, please?

**Review by wiersgallak — APPROVED** [review]
Looks good to me - Thanks!

File: source/overview/architecture.rst [hunk 1]
```diff
@@ -59,6 +59,49 @@ If a WSS connection is not available and HTTPS is substituted, the system will a
...
+
+The following table lists the Mattermost services ports for Mattermost Server, push proxy, and mobile app clients. System Admins with clients that need to speak to the Mattermost Server without a proxy can open specific filewall ports as needed.
+
+**Mattermost Server**
+
++-------------------------------------------------------------+---------------------------------------+-----------------------------------+-----------+------------+---------------------------------------------------------------+
+| Service Name                                                | Config Setting                        | Port (default)                    | Protocol  | Direction  | Info                                                          |
++=============================================================+=======================================+===================================+===========+============+===============================================================+
+| HTTP/Websocket                                              | ServiceSettings.ListenAddress         | 8065/80/443 (TLS)                 | TCP       | Inbound    | External (no proxy) / Internal (with proxy)                   |
```
Comment (sadohert) [inline source/overview/architecture.rst:72]: These ports were right, but I'm testing how to suggest in the docs with Github

```suggestion
| HTTP/Websocket                                              | ServiceSettings.ListenAddress         | 80666/88/445 (TLS)                 | TCP       | Inbound    | External (no proxy) / Internal (with proxy)                   |
```

File: source/overview/architecture.rst [hunk 2]
```diff
@@ -59,6 +59,49 @@ If a WSS connection is not available and HTTPS is substituted, the system will a
...
++-------------------------------------------------------------+---------------------------------------+-----------------------------------+-----------+------------+ Usually this requires port 80 and 443 when running HTTPS.     |
+|                                                             |                                       |                                   |           |            |                                                               |
++-------------------------------------------------------------+---------------------------------------+-----------------------------------+-----------+------------+---------------------------------------------------------------+
+| Cluster (Gossip)                                            | ClusterSettings.GossipPort            | 8074                              | TCP/UDP   | Inbound    | Internal                                                      |
```
Comment (sadohert) [inline source/overview/architecture.rst:76]:
```suggestion
| Cluster (Rumours)                                            | ClusterSettings.GossipPort            | 8074                              | TCP/UDP   | Inbound    | Internal                                                      |
```

Comment (sadohert) [reply source/overview/architecture.rst:76]: I'm testing how to suggest in the docs with Github

**Review by sadohert — COMMENTED** [review]
Garbage edits don't accept

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**